### ISSUE TITLE
It is recommended that context and slash command be supported in inline edit

### ISSUE THREAD

**Issue by felix1982** [continuedev/continue#1506]
### Validations

- [X] I believe this is a way to improve. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) that requests the same enhancement

### requirement
If context is supported in liline edit, I think it can bring the following features:
1) With @codebase, you can generate valuable code anywhere
2) Using @Code or @Files, you can write unit test cases in test files
3).....

It can bring endless possibilities, similar to github copilot workspace or aider

_No response_

**Comment by sestinj** [comment]
@felix1982 This feature is a bit hidden due to limitations of VS Code, but if you press "Enter" before entering any text in the quick input box, it will show you a picker for context providers that you can include

**Comment by sestinj** [comment]
<img width="605" alt="Screenshot 2024-06-18 at 12 28 02 PM" src="https://github.com/continuedev/continue/assets/33237525/a8cf75fa-27be-4dec-a577-2cbf57aede85">

**Comment by felix1982** [comment]
@sestinj   jetbrains doesn't seem to support it,

**Comment by sestinj** [comment]
Yes, this is true for now. So that we can better prioritize, what are the most important things you are hoping to do with this feature?
